src/Money.php

```diff
@@ -606,7 +608,7 @@ public function formatForHumans(?string $locale = null, ?Closure $callback = nul
         $prefix = $this->currency->getPrefix();
         $suffix = $this->currency->getSuffix();
 
-        $formatter = new \NumberFormatter($locale ?: static::getLocale(), \NumberFormatter::PADDING_POSITION);
+        $formatter = new \NumberFormatter(is_null($locale) ? static::getLocale():$locale, \NumberFormatter::PADDING_POSITION);
```

Comment:
Any reason for this change?

Reply:
Static analyzers don't like using non boolean values in if statement to resolve that I changed it like this